main.go

```diff
@@ -49,6 +49,7 @@ import (
 
 var (
 	log                      = logger.Get()
+	l                        = log.WithField("prefix", "main")
```

Comment:
In general, I like this approach to reduce copy paste, and it should be probably applied to rest of modules in future, pls pls name it smth like `mainLog` or similar.